### ISSUE TITLE
Inhibitions when adding exercises

### ISSUE THREAD

**Issue by M123-dev** [wger-project/wger#668]
## Use case

When you add an exercise there are a few ambiguities:

Does the name + description have to be in english, or can I use my native language?

What license should I choose and why should I choose one. Do I make myself liable, will my name be published. 

## Proposal

For the first one a small disclaimer would be enuph and for the license, why do we even need one for that? 
If I want to add a certain type of push-up, it is not mine. I guess it would be best to make the license optional and if nothing is specified, we can license the exercise to all wger contributor. As it is in large open source software projects.

**Comment by rolandgeider** [comment]
Yeah, at the moment this is not really user friendly, streamlining / simplifying the submition process is one the things we have planned for this relase
